plugins/oauth/hooks.js

```diff
@@ -201,8 +201,9 @@ module.exports = Object.assign({}, StoreMixin, {
 
   userLoginSuccess() {
     const redirectTo = RouterUtil.getRedirectTo();
+    const isValidRedirect = RouterUtil.isValidRedirect(redirectTo);
```

Comment:
`isValidRedirect` function expects a `domain` as an argument but we provide the whole thing